src/app/helpers/backbone/view.js

```diff
@@ -80,6 +79,70 @@ class BaseView extends Backbone.View {
         this.regions = new Regions(this.regions, this);
     }
 
+    delegate(eventName, selector, listener) {
```

Comment:
What's this about? I don't see that it's used, and it's not quite event delegation.